### ISSUE TITLE
ERDDAP utility scripts are not executable

### ISSUE THREAD

**Issue by RoryMMMM** [axiom-data-science/docker-erddap#39]
This is a small issue.
 
The utility scripts (dasds.sh, generateDatasetXML.sh  etc) in the erddap folder are not executable. When debugging and having repeated build/deploy cycles it becomes a little irritating to keep making them executable. 

Adding:
`RUN chmod +x /usr/local/tomcat/webapps/erddap/WEB-INF/*.sh`
to the dockerfile would be useful.

**Comment by MathewBiddle** [comment]
You could make a shell script the runs the docker command to execute the script. Similar to:
https://github.com/ioos/erddap-gold-standard/blob/master/GenerateDatasetsXml.sh

**Comment by kwilcox** [comment]
@RoryMMMM Would you mind posting a little more about your workflow so I can understand why these need to be executable? They are shell scripts and can be called by running `bash /path/to/script.sh`.

**Comment by RoryMMMM** [comment]
Sure,

I'm building a tool that automatically creates the xml files for ~800 datasets. Each dataset has different metadata, different structure etc. I create a small xml snippet along with a csv and mount them into the erddap docker container. I have another tool that gathers all the xml snippets and creates a single datasets.xml file. For the most part it works well. Every now and then I run into a couple datasets that don't get loaded in erddap. This is all configured and deployed using a docker-compose file + env file. 

If a dataset doesn't load I exec into the erddap container, run dasds (or generateDatasetXml if it's real bad) and see what the issue is. In initial development I was running the complete build/up/test cycle and it got pretty irritating that the shell scripts don't have execute permissions on them. So each time I had to run chmod +x to get them executable.

Now that the tool is getting more mature I've moved all the dev code into the build step, so I've got my own dockerfile for erddap based off of this one (just with our organisations logo, text, etc baked in) so I'm sorted.
